kratos/solving_strategies/schemes/residual_based_bdf_scheme.h

```diff
@@ -242,66 +244,8 @@ class ResidualBasedBDFScheme
-                     break;
-            case 4 : mBDF[0] =  25.0/(12.0 * delta_time); //coefficient for step n+1 (25/12Dt if Dt is constant)
-                     mBDF[1] = -48.0/(12.0 * delta_time); //coefficient for step n (-48/12Dt if Dt is constant)
-                     mBDF[2] =  36.0/(12.0 * delta_time); //coefficient for step n-1 (36/12Dt if Dt is constant)
-                     mBDF[3] = -16.0/(12.0 * delta_time); //coefficient for step n-2 (16/12Dt if Dt is constant)
-                     mBDF[4] =  3.0/(12.0 * delta_time); //coefficient for step n-3 (3/12Dt if Dt is constant)
-                     break;
-            case 5 : mBDF[0] =  137.0/(60.0 * delta_time); //coefficient for step n+1 (137/60Dt if Dt is constant)
-                     mBDF[1] = -300.0/(60.0 * delta_time); //coefficient for step n (-300/60Dt if Dt is constant)
-                     mBDF[2] =  300.0/(60.0 * delta_time); //coefficient for step n-1 (300/60Dt if Dt is constant)
-                     mBDF[3] = -200.0/(60.0 * delta_time); //coefficient for step n-2 (-200/60Dt if Dt is constant)
-                     mBDF[4] =  75.0/(60.0 * delta_time); //coefficient for step n-3 (75/60Dt if Dt is constant)
-                     mBDF[5] =  -12.0/(60.0 * delta_time); //coefficient for step n-4 (-12/60Dt if Dt is constant)
-                     break;
-            case 6 : mBDF[0] =  147.0/(60.0 * delta_time); //coefficient for step n+1 (147/60Dt if Dt is constant)
-                     mBDF[1] = -360.0/(60.0 * delta_time); //coefficient for step n (-360/60Dt if Dt is constant)
-                     mBDF[2] =  450.0/(60.0 * delta_time); //coefficient for step n-1 (450/60Dt if Dt is constant)
-                     mBDF[3] = -400.0/(60.0 * delta_time); //coefficient for step n-2 (-400/60Dt if Dt is constant)
-                     mBDF[4] =  225.0/(60.0 * delta_time); //coefficient for step n-3 (225/60Dt if Dt is constant)
-                     mBDF[5] = -72.0/(60.0 * delta_time); //coefficient for step n-4 (-72/60Dt if Dt is constant)
-                     mBDF[6] =  10.0/(60.0 * delta_time); //coefficient for step n-5 (10/60Dt if Dt is constant)
-                     break;
-            default : KRATOS_ERROR << "Methods with order > 6 are not zero-stable so they cannot be used" << std::endl;
-        }
-
-        const double tolerance = 1.0e-24;
-        KRATOS_WARNING_IF("ResidualBasedBDFScheme", mOrder > 2 && std::abs(delta_time - previous_delta_time) > tolerance) << "For higher orders than 2 the time step is assumed to be constant. Sorry for the inconveniences" << std::endl;
-
-        // Adding to the process info
-        Vector bdf_vector(mOrder + 1);
-        for (std::size_t i_order = 0; i_order < mOrder + 1; ++i_order)
-            bdf_vector[i_order] = mBDF[i_order];
-        r_current_process_info.SetValue(BDF_COEFFICIENTS, bdf_vector);
+        mpBDFUtility->ComputeAndSaveBDFCoefficients(r_current_process_info);
```

Comment:
Does the utilities compute the coefficients to order greater than 2?

Reply:
[It does](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/time_discretization.cpp#L33-L50)

Reply:
@loumalouomega you mean you want to maintain the warning if time order > 2? Maybe we can move it to the time discretization utility...

Reply:
Yep, that will be fine

Reply:
warning recovered in scheme